simulators/utils.py

```diff
@@ -16,19 +16,42 @@ def checksum(msg):
     bin_sum_fixed_lenght = bin_sum.zfill(8)[-8:]
     # Eventually we should return the one complement
     return int(bin_sum_fixed_lenght, base=2) ^ 0xFF
-    
+
 
 def twos_to_int(binary_string):
+    """Return the integer represented by the given binary string.
+    It is mandatory to pad the binary string to the desired bits
```

Comment:
Maybe you should put this sentence before the `twos_to_int('101'.zfill(8))` example.  It is not clear to me why we should fill the string, so maybe it is also useful to complete the examples with the call to `twos_to_int('101')`, in order to see the difference.